index.html

```diff
@@ -17,22 +17,25 @@
                 <nav>
                     <a href="#home">Home</a>
```

Comment:
I suggest to delete this line, as logo in the header in the left corner have the same functionality